code/modules/client/preference/preferences.dm

```diff
@@ -23,6 +23,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	ROLE_NINJA = 21,
 	ROLE_GSPIDER = 21,
 	ROLE_ABDUCTOR = 30,
+	ROLE_DEVIL = 14
```

Comment:
I feel like you should have to have played on Para for much longer than 2 weeks to play a devil.